src/plone/restapi/services/querystringsearch/configure.zcml

```diff
@@ -10,4 +10,11 @@
     permission="zope2.View"
     />
 
+  <plone:service
```

Comment:
We need to keep the existing one because the Root does not implement IContentish, right?